src/app/screens/Library/components/EmptyLibrary/styles.js

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet } from 'react-native';
+import colors from '@constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  image: {
+    marginVertical: 20,
+    maxHeight: 50,
+    maxWidth: 50
```

Comment:
extract common constant value

Reply:
Done! (I don't like me that approach but, i changed it anyway)